.ruby-version

```diff
@@ -0,0 +1 @@
+2.3.0
```

Comment:
It's a new file all together.  This is something we had talked about doing before.  I'm okay with `.ruby-version` but I'd like to refrain from `.ruby-gemset` because that's specific to RVM.